### ISSUE TITLE
Skip Build Generation for cases that all lines modified for any by the parents are also included on the other parent

### ISSUE THREAD

**Issue by rafaelmotaalves** [spgroup/miningframework#118]
Those cases occur very often, and they don't need to be built and executed because, by definition, those cases won't have conflicts detected.

**Comment by pauloborba** [comment]
@rafaelmotaalves do we already skip cases in which one of the branches just changed whitespace?

**Comment by rafaelmotaalves** [comment]
Yes, we do, because we use the semantical diff (diffj) to detect which lines have changed. @pauloborba
